docs/makedocs.jl

```diff
@@ -172,10 +172,18 @@ function populate_stork_config(deploydecision)
     finally
         cd(wd)
     end
-    cp("__site/libs/stork/config.toml", "__site/libs/stork/config_filled.toml", force = true)
+    cp("__site/libs/stork/config_box.toml", "__site/libs/stork/config_box_filled.toml", force = true)
+    cp("__site/libs/stork/config_page.toml", "__site/libs/stork/config_page_filled.toml", force = true)
 
-    toml = TOML.parsefile("__site/libs/stork/config.toml")
-    open("__site/libs/stork/config_filled.toml", "w") do io
+    toml = TOML.parsefile("__site/libs/stork/config_box.toml")
```

Comment:
This should be a loop if the new section does the same as the old, otherwise it's hard to see if there might be differences (I didn't see any)

Reply:
Yeah, it does the same as the old but for two files. I will make it a loop